src/app/patient/patient.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "../person/person.hpp"
+#include <algorithm>
+#include <memory>
+#include <string>
+#include <unordered_set>
+#include <vector>
+
+enum class Allergent
+{
+    Allergent = 0,
+    AnotherAllergent = 1,
+    SomeAllergent = 2,
+    DifferentAllergent = 3
+};
```

Comment:
Consider separating enum class into different file. (I know its temporary, please make issue out of it)

Reply:
Applied, however PR was not finished yet